### PR TITLE
Remove unused gem and rake task

### PR DESCRIPTION
**Pull request by ryankbales**
Remove existing data rake task and `g5_heroku_app_name_formatter` gem since it's not used outside of that.

NOTE:  THIS CODE IS NOT USED IN THIS APP.  It was only used during the centralization process.

### CODE REVIEW

**Comment by johnlucia** [comment]
:+1:

File: lib/tasks/existing_data.rake [hunk 1]
```diff
@@ -1,134 +0,0 @@
...
-        end
-
-      else 
-        # puts "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX #{url} "
-        bad << client.name
-      end
-    end
-
-    puts "\n\n\n\n\n\n\n\n"
-    puts "BAD:\n\n\n"
```
Comment (thescubageek) [inline lib/tasks/existing_data.rake]: Just lovely code right here...

**Comment by thescubageek** [comment]
:+1:

**Comment by variousred** [comment]
:shipit:

**Pull request merged**